rocksdb/_rocksdb.pyx

```diff
@@ -1784,6 +1785,90 @@ cdef class Options(ColumnFamilyOptions):
         def __set__(self, value):
             self.opts.best_efforts_recovery = value
 
+cdef class TransactionDBOptions(object):
+    cdef transaction_db.TransactionDBOptions* opts
+    cdef cpp_bool in_use
+
+    def __cinit__(self):
+        self.opts = new transaction_db.TransactionDBOptions()
+        self.in_use = False
+
+    def __dealloc__(self):
+        if not self.opts == NULL:
+            del self.opts
+
+    def __init__(self, **kwargs):
+        for key, value in kwargs.items():
+            setattr(self, key, value)
+
+    property max_num_locks:
+        def __get__(self):
+            return self.opts.max_num_locks
+        def __set__(self, value):
+            self.opts.max_num_locks = value
+
+    property max_num_deadlocks:
+        def __get__(self):
+            return self.opts.max_num_deadlocks
+        def __set__(self, value):
+            self.opts.max_num_deadlocks = value
+
+    property num_stripes:
+        def __get__(self):
+            return self.opts.num_stripes
+        def __set__(self, value):
+            self.opts.num_stripes = value
+
+    property transaction_lock_timeout:
+        def __get__(self):
+            return self.opts.transaction_lock_timeout
+        def __set__(self, value):
+            self.opts.transaction_lock_timeout = value
+
+    property default_lock_timeout:
+        def __get__(self):
+            return self.opts.default_lock_timeout
+        def __set__(self, value):
+            self.opts.default_lock_timeout = value
+
+    # TODO property custom_mutex_factory
+    property write_policy:
+        def __get__(self):
+            if self.opts.write_policy == transaction_db.WRITE_COMMITTED:
+                return 'write_committed'
+            if self.opts.write_policy == transaction_db.WRITE_PREPARED:
+                return 'write_prepared'
+            if self.opts.write_policy == transaction_db.WRITE_UNPREPARED:
+                return 'write_unprepared'
+            raise InvalidArgument("Unknown write policy")
```

Comment:
This is much better than raising Exception, but if we decide to continue work on RocksDB, we should create a specific exception hierarchy for errors.

Reply:
The InvalidArgument is from the `errors.py` in this project, so I just reused it.